tracee-rules/signature.go

```diff
@@ -83,19 +84,35 @@ func findRegoSigs(dir string) ([]types.Signature, error) {
 
 	var res []types.Signature
 
+	regoHelpers := []string{regoHelpersCode}
 	for _, file := range files {
-		if filepath.Ext(file.Name()) != ".rego" {
+		if file.Name() == "helpers.rego" {
 			continue
 		}
```

Comment:
seems like this test is included in L93, no?

Reply:
L93 checks if a file is **not** a helper, we don’t process it. But ‘helpers.go’ is a valid helper, but we don’t want to process it, because it is already embedded.

Reply:
right, sorry. 
I think we should not skip helpers.rego actually because it's embedded (that was the motivation to embed it). but I'll live that for a future PR maybe